src/GafferVDB/MeshToLevelSet.cpp

```diff
@@ -184,66 +188,102 @@ const FloatPlug *MeshToLevelSet::interiorBandwidthPlug() const
 	return getChild<FloatPlug>( g_firstPlugIndex + 3 );
 }
 
-bool MeshToLevelSet::affectsProcessedObject( const Gaffer::Plug *input ) const
+bool MeshToLevelSet::affectsMergedObject( const Gaffer::Plug *input ) const
 {
 	return
-		ObjectProcessor::affectsProcessedObject( input ) ||
+		MergeObjects::affectsMergedObject( input ) ||
 		input == gridPlug() ||
 		input == voxelSizePlug() ||
 		input == exteriorBandwidthPlug() ||
 		input == interiorBandwidthPlug()
 	;
 }
 
-void MeshToLevelSet::hashProcessedObject( const ScenePath &path, const Gaffer::Context *context, IECore::MurmurHash &h ) const
+void MeshToLevelSet::hashMergedObject(
+	const ScenePath &path, const Gaffer::Context *context, IECore::MurmurHash &h
+) const
 {
-	ObjectProcessor::hashProcessedObject( path, context, h );
+	MergeObjects::hashMergedObject( path, context, h );
 
 	gridPlug()->hash( h );
 	voxelSizePlug()->hash( h );
 	exteriorBandwidthPlug()->hash ( h );
 	interiorBandwidthPlug()->hash ( h );
 }
 
-IECore::ConstObjectPtr MeshToLevelSet::computeProcessedObject( const ScenePath &path, const Gaffer::Context *context, const IECore::Object *inputObject ) const
+IECore::ConstObjectPtr MeshToLevelSet::computeMergedObject( const std::vector< std::pair< IECore::ConstObjectPtr, Imath::M44f > > &sources, const Gaffer::Context *context ) const
 {
-	const MeshPrimitive *mesh = runTimeCast<const MeshPrimitive>( inputObject );
-	if( !mesh )
```

Comment:
This is a change in behaviour that I hadn't anticipated. In the previous version, non-meshes were passed through, but they are now replaced with an empty level set. I think this is OK (unless anyone else has concerns?) but will need calling out in Changes.md.